docs/_docs/getting-started/deploy-docker.md

```diff
@@ -145,6 +145,9 @@ services:
     # - ALPINE_CORS_EXPOSE_HEADERS=Origin, Content-Type, Authorization, X-Requested-With, Content-Length, Accept, Origin, X-Api-Key, X-Total-Count
     # - ALPINE_CORS_ALLOW_CREDENTIALS=true
     # - ALPINE_CORS_MAX_AGE=3600
+    # 
+    # Optional system property to enable default notification publisher templates override
+    # - JAVA_OPTIONS="-Ddtrack.template.default.override.enabled=true"
```

Comment:
Recommending this override is suboptimal, as it contains important JVM options per default:

https://github.com/DependencyTrack/dependency-track/blob/d00c93317e73a4040f3517f07ad8befda499dd0c/src/main/docker/Dockerfile#L19

Maybe introducing an environment variable would be a better option? Using `System.getenv` in the code should suffice, I fear we can't use Alpine's `Config` here because it can't be extended.

Reply:
Yeah environment variable should be ok. To be consistent, I will introduce two env variables for the enable flag and base directory. Alpine is generic anyway. Trying to introduce a Dependency Track's specific config wouldn't be relevant.